fleet_vehicle_service_activity/models/fleet_vehicle_log_services.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class FleetVehicleLogServices(models.Model):
+    _inherit = "fleet.vehicle.log.services"
+
+    @api.model
+    def scheduler_manage_service_date(self):
+        params = self.env["ir.config_parameter"].sudo()
+        delay_alert_service = int(
+            params.get_param("hr_fleet.delay_alert_service", default=30)
+        )
+        date_today = fields.Date.from_string(fields.Date.today())
+        outdated_days = fields.Date.to_string(
```

Comment:
Why are you converting it to string? It can be included in a domain as datetime without problems since v12.